installers/dhall/launcher.dhall

```diff
@@ -9,17 +9,19 @@
 , nodeTimeoutSec = 60
 , reportServer   = cluster.reportServer
 , walletArgs     = [] : List Text
+, logsPrefix     = os.nodeArgs.logsPrefix
+, tlsPath        = os.nodeArgs.tlsPath
+, x509ToolPath   = os.x509ToolPath
 , nodeArgs =
-    [ "--tlsca",               "tls/ca/ca.crt"
-    , "--tlscert",             "tls/server/server.crt"
-    , "--tlskey",              "tls/server/server.key"
+    [ "--tlsca",               "${os.nodeArgs.tlsPath}/server/ca.crt"
+    , "--tlscert",             "${os.nodeArgs.tlsPath}/server/server.crt"
+    , "--tlskey",              "${os.nodeArgs.tlsPath}/server/server.key"
```

Comment:
@KtorZ, wasn't CSL bumped to a version that no longer accepts these options, but `--tlspath`?

Reply:
To be honest, I have a bit lost track of what's going on with this particular PR. I believe you refer to this: https://github.com/input-output-hk/cardano-sl/pull/2988

in which case, it seems that you're right. Yet I wonder, the PR only affects the launcher but no changes have been done to the node's CLI :thinking:  ?